server/lomas_server/constants.py

```diff
@@ -17,58 +20,33 @@
     SECRETS_PATH = "/usr/lomas_server/secrets.yaml"
 
 
-class ConfigKeys(StrEnum):
-    """Keys of the configuration file"""
-
-    RUNTIME_ARGS: str = "runtime_args"
-    SERVER: str = "server"
-    SETTINGS: str = "settings"
-    DEVELOP_MODE: str = "develop_mode"
-    TIME_ATTACK: str = "time_attack"
-    SUBMIT_LIMIT: str = "submit_limit"
-    DB: str = "admin_database"
-    DB_TYPE: str = "db_type"
-    DB_TYPE_MONGODB: str = "mongodb"
-    MONGODB_ADDR: str = "address"
-    MONGODB_PORT: str = "port"
-    DP_LIBRARY: str = "dp_libraries"
-
-
-class AdminDBType(StrEnum):
-    """Types of administration databases"""
-
-    YAML: str = "yaml"
-    MONGODB: str = "mongodb"
-
-
-class TimeAttackMethod(StrEnum):
-    """Possible methods against timing attacks"""
-
-    JITTER = "jitter"
-    STALL = "stall"
-
+# Misc
+# -----------------------------------------------------------------------------
 
 # Server states
 DB_NOT_LOADED = "User database not loaded"
 CONFIG_NOT_LOADED = "Config not loaded"
 SERVER_LIVE = "LIVE"
 
-
 # General values
 SECONDS_IN_A_DAY = 60 * 60 * 24
 
 # DP constants (max budget per user per dataset)
 EPSILON_LIMIT: float = 10.0
 DELTA_LIMIT: float = 0.01
 
+# Dummy dataset generation
+DUMMY_NB_ROWS = 100
+DUMMY_SEED = 42
+RANDOM_STRINGS = list(string.ascii_lowercase + string.ascii_uppercase + string.digits)
```

Comment:
I think these are not supported anymore:.if categorical string, the strings must be provided

Reply:
i see they are still there actually.. can do later

Reply:
Yes they still are. Was the decision not to support them final? I don't remember.

Reply:
Ah yes, for the case where it is id (right?) not sure anymore

Reply:
Ah yes that makes sense. I guess we will have to decide how to treat these columns.